### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by afmcdnl**
Added the MicroCode line in the visual languages.  

Thanks!

### Resource description

This is a new programming language development from the Microsoft team to encourage young people to code with the micro:bit. 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [ X] I have read and understood the [Contribution Guidelines](https://github.com/carlosperate/awesome-microbit/blob/master/contributing.md)
- [X ] I am making an individual pull request for each suggestion
- [ X] I have used the correct spelling and capitalisation for `BBC micro:bit`, `micro:bit`, and `Micro:bit Educational Foundation` (resource title could be excluded if there is a good reason)
- [ X] The content linked is in English, or it contains an English translation
- [ X] I have added the new entry to the bottom of its the category
- [ X] I have used the following format:
```
- [Resource Title](link) - Resource short Description (2 lines or less in total)
```

### CODE REVIEW

**Comment by afmcdnl** [comment]
Many thanks!

**Comment by afmcdnl** [comment]
Hope this works for Hacktoberfest 2022 - sorry for the multiple comments

File: README.md
Comment (carlosperate) [inline README.md:72]:
```suggestion
- [MicroCode](https://microsoft.github.io/microcode/) - MicroCode is a icon-based (minimal text), editor for the micro:bit V2, which can also be used to write programmes directly on the micro:bit with an Arcade Shield. It is suitable for younger learners and users with variable accessibility needs.
```

**Review by carlosperate — APPROVED** [review]
Awesome, thanks for the submission!
I've tweaked the text slightly, and will merge right away 👍 
Once it's merge it should count to the Hacktoberfest entries.

Thanks again!

**Pull request merged**
